src/appShell/App/Container.tsx

```diff
@@ -49,6 +50,10 @@ export default class Container extends React.Component<IContainerProps, {}> {
         return getBrowserWindow().routingStore;
     }
 
+    private get appStore(){
```

Comment:
@adamabeshouse at some point i would like to use <Provider> to allow child components access to these global stores without resorting to window object.  But i just don't quite understand how it works with typing yet.  Not too important